### PR TITLE
Fix #427

### LINKED ISSUE #427: Add a systemProperties.adoc with a table of system properties

### PR DESCRIPTION
**Pull request by j4sm1ne96**
Create adoc of system properties

### CODE REVIEW

File: systemProperties.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+== System Properties
+Below, a number of relevant System Properties are listed.
+
+.System properties
+[cols=4*, options="header"]
+|===
+| Property Key | Default | Description | Java Variable Name (Type)
+| chronicle.map.creation.debug | `false` | If 'true', displays message regarding aspects of map creation, including what the existing file is and what the name and header of the map is  | _MAP_CREATION_DEBUG_ (boolean)
+| chronicle.map.sparseFile | `false` | Dictates if a file is a sparseFile or not, which is important to know as the fragmented file may hang the program and cause dmsg reports | sparseFile (boolean)
```
Comment (minborg) [inline systemProperties.adoc:9]: Ooops. Is this really the case? Can you check with the person that committed this message? If true, we should not expose this system property.

Comment (j4sm1ne96) [reply systemProperties.adoc:9]: @glukos I believe that you added information about this system property, I will add you as a reviewer

Comment (glukos) [reply systemProperties.adoc:9]: I think we should ask @peter-k-lawrey - judging by the history, we added `chronicle.map.sparseFile=true` to `Chronicle-Map\system.properties` to test sparse files for a while. Probably the comment is related to that fact. 
+1 to not expose if we are not sure about it yet.

**Comment by peter-lawrey** [comment]
This feature is confusing as there appears to be an assumption that if sparseFile=false that this turns off sparse files.
It should probably have been named, use aggressive sparse file-based sizing if available.
It needs further reviewed before we expose this sort of functionality, as many find sparse files are a bit magical, though using this approach improves performance.

**Pull request closed without merging**